validation/README.md

```diff
@@ -0,0 +1,116 @@
+# Validation specifications
+
+Data validation allows users to check whether their data meets the ODM dictionary format by reporting what variables and entries either meet or do not meet the ODM format.
+
+  
+Validation can be performed throughout the data life cycle. Measurement laboratories can validate their data before sharing data with repositories or other researchers. Data repositories can validate data when they accept data from laboratories. Public health epidemiologists and modellers can validate data before analysis. 
+
+  
+The emphasis of ODM validation is on providing guidance and support. When you validate data, you receive a report with warnings and errors. ODM validation emphasizes warnings rather than errors. An example of a warning is when an email does not contain an '\@' symbol. An error is when a unique identifier is not repeated (it is not unique). Where possible, data is coerced to validate data. For example, ODM will transform data into real numbers if data is entered as text. (\`temperature\` = \`'1'\` will be transformed into \`temperature\` = \`1\`.
+
+## Context
+
+1.  Data validation must be performed for/by a range of data custodians. Many different data custodians generate wastewater surveillance data, including academic testing labs, national and commercial testing labs, municipal wastewater utilities and public health. These data custodians require different types of support to validate the data they generate in ODM format.
+
+2.  Data validation must be performed on different platforms. Data custodians have many different data systems. Most academic testing labs use Excel. Many data custodians do not use open programming software such as R or Python. ArcGIS and Microsoft 365 are used by key stakeholders. An example of a platform-specific validation is missing data. The ODM format for missing data is to leave the value blank. However, different program platforms specific format for missing data to facilitate analyses. Missing data formats vary considerably between platforms. Missing data can also be coded in different formats within a programming language. The use of platform-specific missing data tokens allows users of the platform to use a consistent missing data format.
```

Comment:
Don't think this line is needed, `However, different program platforms specific format for missing data to facilitate analyses`

Reply:
Took me some time but I think you're advocating for most of the validation features to be platform-agnostic?

Reply:
I'm also not sure what this point suggests. Do we mean that values should be parsed from platform-native to ODM before being validated? Or that validated data should be converted to another format?